kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/MetricsST.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests;
+
+import java.time.Duration;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.strimzi.api.kafka.model.Kafka;
+
+import io.kroxylicious.systemtests.extensions.KroxyliciousExtension;
+import io.kroxylicious.systemtests.installation.kroxylicious.Kroxylicious;
+import io.kroxylicious.systemtests.metrics.MetricsCollector;
+import io.kroxylicious.systemtests.resources.ComponentType;
+import io.kroxylicious.systemtests.steps.KafkaSteps;
+import io.kroxylicious.systemtests.steps.KroxyliciousSteps;
+import io.kroxylicious.systemtests.templates.metrics.ScraperTemplates;
+import io.kroxylicious.systemtests.templates.strimzi.KafkaNodePoolTemplates;
+import io.kroxylicious.systemtests.templates.strimzi.KafkaTemplates;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+import static io.kroxylicious.systemtests.utils.MetricsUtils.assertMetricValue;
+import static io.kroxylicious.systemtests.utils.MetricsUtils.assertMetricValueCount;
+import static io.kroxylicious.systemtests.utils.MetricsUtils.assertMetricValueHigherThan;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+/**
+ * @description This test suite is designed for testing metrics exposed by kroxylicious.
+ */
+@ExtendWith(KroxyliciousExtension.class)
+public class MetricsST extends AbstractST {
+    private static final Logger LOGGER = LogManager.getLogger(MetricsST.class);
+    private final String clusterName = "my-cluster";
+    protected static final String BROKER_NODE_NAME = "kafka";
+    private static final String MESSAGE = "Hello-world";
+    private static Kroxylicious kroxylicious;
+    private MetricsCollector kroxyliciousCollector;
+
```

Comment:
You've tested just a couple of the metrics exposed by Kroxy.  I think this is good enough for now.

Reply:
Me too